src/CalcViewModel/Common/LocalizationSettings.h

```diff
@@ -15,9 +15,111 @@ namespace CalculatorApp
         private:
             LocalizationSettings()
                 // Use DecimalFormatter as it respects the locale and the user setting
+                // CSHARP_MIGRATION: TODO:
                 //: LocalizationSettings(LocalizationService::GetInstance()->GetRegionalSettingsAwareDecimalFormatter())
             {
-                LocalizationSettings(LocalizationService::GetInstance()->GetRegionalSettingsAwareDecimalFormatter());
+                Windows::Globalization::NumberFormatting::DecimalFormatter ^ formatter =
```

Comment:
what about we put the content between line21 to line122 into a private method like `Initialize()` to avoid the duplicated code snippet?

Reply:
Done